src/clj_momo/lib/es/document.clj

```diff
@@ -263,10 +287,10 @@
    params :- s/Any]
   (let [es-params (generate-es-params query params)
         res (safe-es-read
-             (client/post
+             (client/get
```

Comment:
It's not an error.

> Both HTTP GET and HTTP POST can be used to execute search with body. Since not all clients support GET with body, POST is allowed as well.
Ref: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-body.html